sources/packages/backend/libs/src-sql/disbursement-overawards/Bulk-update-bcsl-disbursement-overaward.sql

```diff
@@ -0,0 +1,12 @@
+/*
+ * Update BCSL overawards from SFAS integration to disbursement overawards table.
+ */
+UPDATE	sims.disbursement_overawards disbursement_overawards
+SET		overaward_value = sfas_individuals.bcsl_overaward, 
+		updated_at = NOW(),
+		modifier = users.id
+FROM 	sims.sfas_individuals sfas_individuals
+		INNER JOIN sims.users users ON users.last_name = 'system-user'
+WHERE 	disbursement_overawards.student_id = sfas_individuals.student_id
+AND		disbursement_overawards.origin_type = 'Legacy overaward'
+AND		disbursement_overawards.disbursement_value_code = 'BCSL';

```

Comment:
We can add also add a clause to avoid updating the value if the value is the same.
The sfas_individuals can be updated multiple times due to other columns changes.